lib/cdk-infra-stack.ts

```diff
@@ -501,14 +514,37 @@ export default class FhirWorksStack extends Stack {
             description: 'FHIR API Server',
             entry: path.join(__dirname, '../src/index.ts'),
             handler: 'handler',
-            bundling: defaultLambdaBundlingOptions,
+            bundling: {
+                ...defaultLambdaBundlingOptions,
+                commandHooks: {
+                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                    beforeBundling(inputDir, outputDir) {
+                        return [];
+                    },
+                    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                    beforeInstall(inputDir, outputDir) {
+                        return [];
+                    },
+                    afterBundling(inputDir, outputDir) {
+                        // copy all the necessary files for the lambda into the bundle
+                        // this allows the lambda functions for bulk export to have access to these files within the lambda instance
```

Comment:
Update the comment for implementation guide

Reply:
Thanks for catching this- updated.